### ISSUE TITLE
Support MacOS  NSAccessibility 

### ISSUE THREAD

**Issue by reagle** [zim-desktop-wiki/zim-desktop-wiki#2292]
Hello, on MacOS I use the Moom window manager with which Zim does not work. In the image, you can see that when I hit the shortcut to move a window I only see a no/prohibition symbol.

![moom-zim](https://user-images.githubusercontent.com/596814/211165562-76bec4ac-7060-42f5-82fa-e8daacffdff2.png)

The Moom developer says they rely upon support of the [NSAccessibility](https://developer.apple.com/documentation/appkit/nsaccessibility) API. Is this available in GTK4? Is this something that Zim could do?

**Comment by dehesselle** [comment]
Sorry for the bad news, but not going to happen. There are no backends for non-Linux platforms available. See here: https://discourse.gnome.org/t/question-gtk3-and-gtk4-how-broken-is-accessibility-on-non-linux-platforms/5981

**Comment by reagle** [comment]
Does [this documentation](https://docs.gtk.org/gtk4/section-accessibility.html#implementations) imply it would be available via GTK4?

**Comment by dehesselle** [comment]
No, that documentation simply states what needs to be implemented to accommodate GTK's needs to make their API work. The phrasing is "specification style", so if you're not used to it, it can be misunderstood.

**Comment by reagle** [comment]
For reference, here's an additional link about MacOS accessibility: https://developer.apple.com/documentation/accessibility/integrating_accessibility_into_your_app
